lxc/move.go

```diff
@@ -1,33 +1,22 @@
 package main
 
 import (
+	"github.com/codegangsta/cli"
```

Comment:
Insert a new line after this import to keep it separate from the github.com/lxc/lxd/* ones.